package com.generation.vhfc.ManejoExcepciones;

public class NoComprobadasExceptions {
public static void main(String[] args) {
	
	try {
		 
		int division = 10 / 0;
		System.out.println("---> "+ division);

	}catch (ArithmeticException e ) {
		System.out.println("capturando la exception " + e.getMessage());
	}
	
}
}
